CMakeLists.txt

```diff
@@ -1,13 +1,17 @@
 cmake_minimum_required(VERSION 3.5)
 project(cppJNI)
+set(CMAKE_CXX_STANDARD 14)
 
+# Extend CMAKE_PREFIX_PATH with HanaConfig.cmake and Gtest root
+#list(APPEND CMAKE_PREFIX_PATH "")
+#SET(GTEST_ROOT "")
 enable_testing()
 
-find_package(Boost 1.61 REQUIRED)
-
+find_package(Threads REQUIRED)
+find_package(Hana REQUIRED)
 add_definitions(-DUSE_TYPESTRING=8)
 
-add_compile_options(-std=c++14 -ftemplate-depth-1024 -ftemplate-backtrace-limit=0 -Wno-gnu-string-literal-operator-template)
+add_compile_options(-ftemplate-depth-1024 -ftemplate-backtrace-limit=0 -Wno-gnu-string-literal-operator-template)
```

Comment:
cppJNI shouldn't compile without C++14 features, so, if you remove this parameter, please provide different way of forcing standard (I know, I should use CMake macros for that : /, but still).

Reply:
I already did it couple lines before.
"set(CMAKE_CXX_STANDARD 14)" This is general way to do it.

Reply:
Oh, sorry, missed that : )!